input/resources/au-core-medicationrequest.xml

```diff
@@ -121,6 +121,11 @@
       </extension>
       <path value="MedicationRequest.medication[x].coding"/>
       <sliceName value="pbs"/>
+      <max value="1" />
```

Comment:
Max  value of 1 is not appropriate for the terminology slices in AU Core Medication and AU Core MedicationRequest - AU Base does not restrict to be max of 1.

Reply:
Fixed